src/DataGrid.tsx

```diff
@@ -992,15 +992,19 @@ function DataGrid<R, SR, K extends Key>(
       const gridRowStart = headerAndTopSummaryRowsCount + rowIdx + 1;
       if (isGroupRow(row)) {
         ({ startRowIndex } = row);
-        const isGroupRowSelected =
```

Comment:
![image](https://github.com/adazzle/react-data-grid/assets/1452717/5f263b71-dba8-4abf-801f-f44ca2a2fc30)

this fails on the grouping demo if `rowKeyGetter` is not specified